internal/validations/clusterlogforwarder/validate_clusterlogforwarderspec.go

```diff
@@ -330,7 +330,8 @@ func verifyOutputURL(output *loggingv1.OutputSpec, conds loggingv1.NamedConditio
 			// Some output types allow a missing URL
 			// TODO (alanconway) move output-specific valiation to the output implementation.
 			if output.Type == loggingv1.OutputTypeCloudwatch ||
-				output.Type == loggingv1.OutputTypeGoogleCloudLogging || output.Type == loggingv1.OutputTypeLoki {
+				output.Type == loggingv1.OutputTypeGoogleCloudLogging ||
+				(output.Type == loggingv1.OutputTypeLoki && output.Name == loggingv1.OutputNameDefault) {
```

Comment:
@vparfonov This validates the opposite of what the observed issue.  Loki requires a URL; there is nothing in the client that crafts one.  I believe the issue is a result of a missing URL

Reply:
> @vparfonov This validates the opposite of what the observed issue. Loki requires a URL; there is nothing in the client that crafts one. I believe the issue is a result of a missing URL

Yes, and URL will be added automatically for `default` output: https://github.com/openshift/cluster-logging-operator/blob/master/internal/logstore/lokistack/logstore_lokistack.go#L220

Reply:
@vparfonov my apologize for the delay in response but this change will not work for loki regardless if it is the default or not.  We do not require the "default" loki to be named "default", only prefixed with it.  Please think about the reconciliation process:

* [migrate](https://github.com/openshift/cluster-logging-operator/blob/release-5.8/internal/migrations/clusterlogforwarder/migrate_clusterlogforwarder.go#L54)
* validate
* reconcile

This means the validate code should fail if  URL is not defined, which is what is reported. "Default" is a special case which we remove in reconciliation processing by "migrating" the CLF to have a proper definition for our internal loki